README.md

```diff
@@ -12,9 +12,12 @@ The main purpose of this client is to be a desktop application fast and simple t
 - Qt min version 6.0.0
 - CMake min version 3.16.3
 
+```bash
+cmake -S . -B build && cmake --build build 
```

Comment:
This is not a substitute for other instruction builds...

Reply:
Are you sure? Should be the same but just in one command line.

Reply:
I prefer to keep simple as possible for new developer :)

Reply:
So better split the two command?
Like:
```
cmake -S . -B build
cmake --build build 
```

Reply:
Yes, better